doc/contributing/docs/images.rst

```diff
@@ -0,0 +1,84 @@
+Images
+======
+
+Images are useful in explanations of concepts and structures.
+When you introduce a term or describe a structure of multiple interconnected parts
+(such as a cluster), consider illustrating it with a diagram. If you explain how to
+use a GUI, check if a screenshot can make the doc clearer.
+
+Note that illustrations should complement the text, not replace it. Even with an image,
+the text should be enough for readers to understand the topic.
+
+Don't overuse images: they are harder to support than text. Use them only if they bring
+an obvious benefit.
+
+Diagrams
+--------
+
+There is a basic set of diagram elements – blocks, arrows, and other – to use in Tarantool docs.
+It is stored in `this Miro board <https://miro.com/app/board/uXjVPbCIj_g=/>`_. It also provides
+basic rules for creating diagrams.
+
+Size
+~~~~
+
+There are two sizes of diagram elements:
+
+*   **M** – bigger elements for usage in diagrams with small number of elements.
+*   **S** – smaller elements for usage in diagrams with big number of elements.
+
+Avoid changing the sizes of diagram elements unless it's absolutely necessary.
+
+The diagrams should have the same width. This guarantees that their elements have the same
+size on pages. The examples in the Miro board have frames of the right width.
+Copy the frame and and place your diagram in it without changing its width.
```

Comment:
```suggestion
Copy the frame and and place your diagram in it without changing the frame width.
```
Otherwise it's ambiguous: frame width or diagram width?